examples/example-subgraph/src/types/schema.ts

```diff
@@ -2,12 +2,50 @@ import {
   TypedMap,
   Entity,
   Value,
+  ValueKind,
+  store,
   Address,
   Bytes,
   BigInt
 } from "@graphprotocol/graph-ts";
 
 export class ExampleEntity extends Entity {
+  constructor(id: string) {
+    this.set("id", Value.fromString(id));
+    return this;
+  }
+
+  save(): void {
+    let id = this.get("id");
+    assert(id !== null, "Cannot save ExampleEntity entity without an ID");
+    assert(
+      id.kind == ValueKind.STRING,
+      "Cannot save ExampleEntity entity with non-string ID"
```

Comment:
I think we could add a helper line to the text. Like: "Consider using toHex() to change the Ethereum type to a string"